CoreFoundation/Base.subproj/module.private.modulemap

```diff
@@ -0,0 +1,8 @@
+framework module CoreFoundation_Private [extern_c] [system] {
```

Comment:
This might pose a problem for the merge back process

Reply:
I'm gonna split this up into a separate change and try to handle the libdispatch related clean up before attempting to figure out how to deal with the private headers.

Reply:
sounds like a good idea to me